time/src/main/java/net/jqwik/api/time/DaysOfWeekArbitrary.java

```diff
@@ -0,0 +1,12 @@
+package net.jqwik.api.time;
+
+import java.time.*;
+
+import net.jqwik.api.*;
+
+public interface DaysOfWeekArbitrary extends Arbitrary<DayOfWeek> {
```

Comment:
Since DayOfWeek is just an enum the whole arbitrary class is not necessary or is it. Could be replaced with Arbitraries.of(DayOfWeek.class) I think. only(..) doe not pull its weight either since Arbitraries.of(MONDAY, WEDNESDAY) is just as short.

Reply:
Okay, I'll remove it